tla/consensus/MCAliases.tla

```diff
@@ -0,0 +1,124 @@
+----- MODULE MCAliases -----
```

Comment:
Are those not just Aliases, in general, usable on MC/SIM/Trace?

Reply:
Yes, I just aimed to clarify that the aliases for TLC. Historically, the community used the MC* to designate files specific to TLC -- all conventions suck.